readme-asana.md

```diff
@@ -0,0 +1,67 @@
+# [Taskwarrior](https://taskwarrior.org/) ⬄ [Asana](https://www.asana.com)
+
+## Description
+
+Synchronize Asana tasks.
+
+Upon execution, `tw_asana_sync` will synchronize, and on subsequent runs of the
+program keep synchronized, the following attributes:
+
+- Asana task `name` <-> TW task `description`
+- Asana task `completed` <-> TW task `status`
+- Asana task `due_at` or `due_on` <-> TW task `due`
+- Asana task is deleted <-> TW task is deleted
+
+Current limitations:
+
+- Does not sync Asana tags, project names, subtasks, projects, likes, etc.
+- Does not sync Taskwarrior tags, projects, annotations, etc.
```

Comment:
this is probably outdated? Syncing taskwarrior tags works fine for me

Reply:
Oh really? That's cool!

Reply:
I tried this and it seemed to me like tags were not synced. I added tags to an Asana task, did an import, and they didn't show up on the TW side.

Reply:
Sorry, probably I didn't express this right. I meant that you can use `-t <some-tw-tag` to only sync tasks of a certain TW tag with Asana.

```
Usage: tw_asana_sync [OPTIONS]

Options:
  --token, --token-pass-path TEXT
                                  Path in the UNIX password manager to fetch
  -w, --asana-workspace-gid TEXT  Asana workspace GID used to filter tasks
  -W, --asana-workspace-name TEXT
                                  Asana workspace name used to filter tasks
  --list-asana-workspaces         List the available Asana workspaces
  -t, --taskwarrior-tags TEXT     Taskwarrior tags to sync <----
```

Isn't this supported ?